Src/SDK/Common/Temporal.Serialization/public/ProtobufPayloadConverter.cs

```diff
@@ -0,0 +1,88 @@
+using System;
```

Comment:
I added it. Then I removed it. Then I checked the Java sdk, and I saw it was there, so I went ahead put it back. I found it in the go sdk here https://github.com/temporalio/sdk-go/blob/0fa6692bd57e4db9fee7d15a980a74cea5f8fcae/converter/proto_payload_converter.go and in the java sdk here https://github.com/temporalio/sdk-java/blob/ed3c65f7e3c4a8ed805d1262eee3d17b6dab7b1a/temporal-sdk/src/main/java/io/temporal/common/converter/ProtobufPayloadConverter.java . I think I interpreted it correctly.

Reply:
You are correct. Wow, looks like I did not sync my local copy of Java SDK since this was added in January :)
https://github.com/temporalio/sdk-java/commit/ed3c65f7e3c4a8ed805d1262eee3d17b6dab7b1a

@lorensr, you made that change? Can you please educate me on the background? I read the PR description, but I am not fully clear under what circumstance we end up doing non-json proto serialization. And, if we do it sometimes, why do we not do it always? I realize, other languages have legacy, but new ones could prefer the binary serialization and only use json if the respective payloads arrive. Wouldn't it be more efficient?

Reply:
@mightyshazam thanks for discovering it. Let's leave it. Depending on what we learn from @lorensr, we can decide on the priority order between the two serializers.

Reply:
Default converters have this order—ProtoJSON is before ProtoBinary, so JSON is always used. 

https://github.com/temporalio/sdk-go/blob/5e5645f0c550dcf717c095ae32c76a7087d2e985/converter/default_data_converter.go#L28

We may change the default after displaying binary is supported in web & tctl. You can follow https://github.com/temporalio/ui-server/issues/79